### ISSUE TITLE
Bug stuck at "Reach the airfield" mission.

### ISSUE THREAD

**Issue by salaham-1230** [openmoh/openmohaa#403]
**Describe the bug**
The jeep is stuck at the wooden barrier. The wooden barrier is not responding to the firing.

**To Reproduce**
1. Play the "Reach the airfield" mission.
4. Wait for the wooden barrier to arrive.

**Expected behavior**
The wooden barrier should blown up on firing and game should continue with a clear path.

**Screenshots**
[Screenshot](https://i.postimg.cc/Wz6k6Ykf/aaaaaa.png)

**Logs**
```
^~^~^ Path not found in 'Actor::MoveToPatrolCurrentNode' for (entnum 28, radnum 717, targetname '') from (-3280.000000 -2258.000000 19.156315) to (-2588.703857 -3520.817139 52.614010)
Reason: couldn't find end node
LOCALIZATION ERROR: 'OpenMoHAA console version 0.70.0' does not have a localization entry
PRINT is unbound, use controls menu to set.
PRINT is unbound, use controls menu to set.
Forward velocity: -51.519
(global/spawner.scr, 110):
local.dude.disguise_accept_thread = local.spawner.disguise_accept_thread
local.dude.disguise_accept_thread ^
^~^~^ Script Error : ScriptThreadLabel::SetThread bad label type 'none'

(global/spawner.scr, 116):
local.dude.alarmthread = local.spawner.alarmthread
local.dude.alarmthread ^
^~^~^ Script Error : ScriptThreadLabel::SetThread bad label type 'none'

^~^~^ Path not found in 'Actor::MoveToPatrolCurrentNode' for (entnum 22, radnum 716, targetname '') from (-3296.000000 -3162.000000 51.602444) to (-2588.522461 -3520.797119 52.430939)
Reason: couldn't find end node
```
The last error repeats.

**Version information:**
 - OS: Win
 - OpenMoHAA Version: openmohaa-v0.70.0-windows-x86

**Comment by smallmodel** [comment]
This was fixed in commit 0840cdd83ab0534abbae1198d33f7e379e307f5e - a release with the latest commit can be obtained in GitHub actions https://github.com/openmoh/openmohaa/actions (Build branch)

**Comment by SilentMRG** [comment]
I just got the same softlock on v0.70.0-windows-x64. I use it on Windows 7, just so you know.
